recipes/catch2/2.x.x/LICENSE.txt

```diff
@@ -0,0 +1,23 @@
+Boost Software License - Version 1.0 - August 17th, 2003
```

Comment:
You don't need this since it's available in source_folder/Catch/LICENSE.txt

Reply:
Just to clarify: All recipes in this repository are supposed to be licensed under MIT

The license of the packages software should be mentioned via the `license` attribute within the recipe and a copy of this license should be copied in the build package

Reply:
So, use `source_folder/Catch/LICENSE.txt` in `exports` instead?

Reply:
Yes, so you don't need to export anything, only copy from source_subfoder